data/mods/Aftershock/mobs/robots.json

```
@@ -501,8 +502,8 @@
     "id": "mon_milbot_556",
     "type": "MONSTER",
     "copy-from": "mon_milbot_base",
-    "name": "military robot",
-    "description": "A military robot still operating due to its internal power core.  This one is armed with an electric prod and an integrated 5.56mm firearm.",
+    "name": "Wraitheon Imaginifer",
+    "description": "The Imaginifer is one of the few example of the true humaniform robots; manufactured by Wraitheon for ESCA militaries to compliment infantry soldiers.  While they are based on humaniform schematics, they are far removed from the ENIAC's Famulus. The Wraitheon humaniforms are designed to take full advantage of a humanoid shape, not to look human; nor designed for sapience, but for game theory.",
```

```suggestion
    "description": "The Imaginifer, one of the few examples of true humaniform robots, was manufactured by Wraitheon for ESCA militaries. They are designed to complement infantry soldiers, being able to use their gear and transports.  While similar in shape, they are far removed from the ENIAC's Famulus. The Wraitheon humaniforms are designed to take full advantage of a humanoid shape, not to look human; programmed not for sapience, but for game theory.",
```
What exactly is being meant by "humaniform", other than humanoid and (I'm guessing) human-sized?

---

Humaniform is a term taken from Isaac Asimov's Robot novels, in the robot novels, it references an advanced robot that is indistinguishable from human; however in aftershock lore, it refers to a humanoid robot that uses the human form, but isn't AIMED towards looking human (like the butler bot by ENIAC)

Its optimized to use the human shape to best suit its goals, and thus looks very uncannily, and its programmed to maximize tactical advantage, not sapience, which makes it very unsettling to talk to.